CHANGELOG.md

```diff
@@ -1,13 +1,32 @@
-# dbt_pendo_source version.version
+# dbt_pendo_source v0.6.0
+
+[PR #29](https://github.com/fivetran/dbt_pendo_source/pull/29) includes the following updates:
+
+## Breaking Change for dbt Core < 1.9.6
+> *Note: This is not relevant to Fivetran Quickstart users.*
```

Comment:
```suggestion
> *Note: This is not relevant to Fivetran Quickstart users.*

```
I think we need a line break because it's rendering like this
<img width="902" alt="image" src="https://github.com/user-attachments/assets/ff95e4d0-a4fb-4c9a-8b85-e7fae9e236ac" />

Reply:
Thanks for catching